### ISSUE TITLE
Multiple groups visible/enabled at the same time

### ISSUE THREAD

**Issue by NicoForce** [Drive4ik/simple-tab-groups#970]
**Is your feature request related to a problem? Please describe.**
Tabs hide whenever I change group, I have a main/default group I need to have open at all times while the other groups are optional.

**Describe the solution you'd like**
I would like to have multiple groups visible at the same time (at least 2 groups)

**Comment by abdo2048** [comment]
Yes, it would be nice if there is option to group tabs in one window like the deprecated native firefox tab group.

**Comment by nekohayo** [comment]
In theory you can already do this, I suspect. STG's description mentions:

> - Now fully supports multiple windows

and:

> Simple Tab Groups works across browser instances/windows too. If you select a group in another window, the selected window will jump to the foreground with the chosen group selected. You can even select the specific tab within that group in background browser windows.
> 
> ![GIF example](https://user-images.githubusercontent.com/7843031/33828871-806ccf6e-de76-11e7-9a0e-1ddfb97e878d.gif)

...but it's still a bit unclear to me how it is supposed to work (and how reliably) for popping a group into a new window, and or "merging" it back later to go back to a single window, and how switching between groups behaves when you have multiple windows open 🤔️
